corehq/apps/reports/tasks.py

```diff
@@ -370,9 +370,9 @@ def _expose_download(fpath, use_transfer, zip_name, download_id, num_forms):
 
 
 def _format_filename(form_info, question_id, extension, case_id_to_name):
-    filename = u"{}-user_{}-form_{}{}".format(
+    filename = u"{}-{}-form_{}{}".format(
```

Comment:
decided to drop the `user_` prefix (i had added it a week or two ago) because `user_brudolph@` is confusing and weird. it was useful when it was an ID.